pywbem/_cim_types.py

```diff
@@ -655,11 +659,22 @@ def get_local_utcoffset():
         A positive value indicates minutes east of UTC, and a negative
         value indicates minutes west of UTC.
         """
-        utc = datetime.utcnow()
-        local = datetime.now()
-        if local < utc:
-            return - int(float((utc - local).seconds) / 60 + .5)
-        return int(float((local - utc).seconds) / 60 + .5)
+        if six.PY2:
+            # Effective Python 3.12, utcnow() generates deprecation warning
+            # because utcnow returns a naive datetime object. timezone does
+            # not exist for python 2.7; use utcnow(timezone.utc).
+            with warnings.catch_warnings():
+                warnings.filterwarnings(
+                    "ignore",
+                    message=".*Passing None has been deprecated.")
```

Comment:
Disabling the warnings is not needed because this code is executed only on Python 2 where the warning was not issued yet.

Reply:
agreed done